commit 7d8f5336f6a6a2ab175e3bc9aaad58d657b230ae
Author: Zlatko Čalušić <zcalusic@bitsync.net>
Date:   Sat Mar 4 11:44:44 2023 +0100

    Stop log spam

diff --git a/src/dyn_client.c b/src/dyn_client.c
index 3b67e2c..ee978a3 100644
--- a/src/dyn_client.c
+++ b/src/dyn_client.c
@@ -94,7 +94,7 @@ static void client_unref_internal_try_put(struct conn *conn) {
   }
   ASSERT(conn->owner != NULL);
   conn_event_del_conn(conn);
-  log_warn("%s unref owner %s", print_obj(conn), print_obj(conn->owner));
+  log_debug(LOG_VVERB, "%s unref owner %s", print_obj(conn), print_obj(conn->owner));
   conn->owner = NULL;
   dictRelease(conn->outstanding_msgs_dict);
   conn->outstanding_msgs_dict = NULL;
