commit 016ff64acc005a15ca5812e32f885b7e26e78ad7
Author: Zlatko Čalušić <zcalusic@bitsync.net>
Date:   Sat Mar 4 12:33:56 2023 +0100

    Fix memory leak

diff --git a/src/dyn_proxy.c b/src/dyn_proxy.c
index 37220ed..3617588 100644
--- a/src/dyn_proxy.c
+++ b/src/dyn_proxy.c
@@ -172,7 +172,7 @@ static rstatus_t proxy_accept(struct context *ctx, struct conn *p) {
     return DN_ENOMEM;
   }
   c->sd = sd;
-  string_copy_c(&c->pname, (unsigned char *)dn_unresolve_peer_desc(c->sd));
+  /* string_copy_c(&c->pname, (unsigned char *)dn_unresolve_peer_desc(c->sd)); */
 
   stats_pool_incr(ctx, client_connections);
 
